localization/english/event_localization/wc_central_kalimdor_events_l_english.yml

```diff
@@ -0,0 +1,250 @@
+cen_kal.1062.a:0 "Prepare the final feast, my kin..."
+cen_kal.1062.b:0 "Take the walk on my own..."
+cen_kal.1062.c:0 "Perhaps it’s best to wait..."
+
+cen_kal.1063.desc:0 "I have summoned my kin and my warlords for one final time, to share a feast of blood and glory. They have been fierce and loyal to me, aiding me in my conquests and following me in my wars. They have also been my clan and my comrades, standing by me in my troubles and celebrating with me in my victories.\n\nI look at their faces, some scarred and hardened, some eager and restless, and I feel a burst of pride and respect. I raise my cup and toast to them, to Agamaggan, and to our tribe."
+cen_kal.1063.a:0 "It is time..."
+cen_kal.1063.a.tt:0 "Your time has come, you pack a weapon and head to the wilderness seeking a worthy death in battle"
+cen_kal.1063.b:0 "Perhaps it’s best to wait..."
+
+cen_kal.1064.desc:0 "I have left everything behind, walking alone into the wilderness. I have no destination, no purpose, I only seek a worthy end, a final challenge, a glorious death. I wander for hours, feeling the wind on my face, the sun on my back, the earth under my feet. I see no signs of life, no animals, no enemies. The land is barren and desolate, as if the spirits have forsaken me.\n\nThen I hear it. A roar that shakes the ground, a snarl that chills my blood, a growl that fills my ears. I turn and see a massive beast, a creature twice my size. It is the most fearsome thing I have ever seen, and it is looking right at me. It senses my weakness, my frailty, my insignificance. It sees me as prey, as food, as nothing.\n\nBut I am not nothing. I will not die like a coward, like a slave. I will die like a leader. I grip my weapon tightly, raise my standard proudly, and face the beast with all my might."
+cen_kal.1064.a:0 "My moment is finally here, for Agamaggan!"
+cen_kal.1064.b:0 "It is not too late!"
+
+centaur_empire_name:0 "Maraudor"
+cen_kal.1065.name:0 "Maraudor Beckons"
+cen_kal.1065.desc:0 "It has been ages since the centaur last ruled Kalimdor, the tauren and the quilboar have encroached on our lands and challenged our might. They are weak and cowardly, they have no courage, no glory. They only know how to cheat and scheme. They are always plotting against us, always trying to take what is ours by right.\n\nAfter today, no more. I have crushed the old khanates, reclaimed our old domain, and made the centaur supreme!"
```

Comment:
```suggestion
cen_kal.1065.desc:0 "It has been ages since the centaurs last ruled Kalimdor, the tauren and the quilboar have encroached on our lands and challenged our might. They are weak and cowardly, they have no courage, no glory. They only know how to cheat and scheme. They are always plotting against us, always trying to take what is ours by right.\n\nAfter today, no more. I have crushed the old khanates, reclaimed our old domain, and made the centaur supreme!"
```